context/app/static/js/helpers/queries.ts

```diff
@@ -35,9 +35,16 @@ export const includeDatasetsAndImageSupports: QueryDslQueryContainer = {
   bool: {
     should: [
       {
-        terms: { entity_type: ['Dataset', 'Support'] },
```

Comment:
I think just appending the `.keyword` to the field would work here as well?

Reply:
That approach leads to non-dataset support descendants being pulled in, e.g. publication ancillary data, since it matches against the `Support` entity type clause:

![image](https://github.com/user-attachments/assets/c6e80d11-f132-48c1-b784-5206b0ce7dad)

With the separate `entity_type.keyword` clauses, we ensure that the returned entities are either:
- a dataset
- an image pyramid support

![image](https://github.com/user-attachments/assets/83f47779-528c-464f-8d1b-636dd50f7fe4)